modules/kmm-using-tolerations-for-kernel-module-scheduling.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="kmm-using-tolerations-for-kernel-module-scheduling_{context}"]
+= Using tolerations for kernel module scheduling
+
+There are circumstances where you will need to evacuate workloads on a node before upgrading a kernel module, which you can do through taints. 
+A taint allows a node to refuse a pod to be scheduled unless that pod has a matching toleration.
+
+However, users also need to set tolerations to allow KMM pods that run housekeeping operations, such as kernel module upgrades. The tolerations must match the taint that gets added to the nodes.
+
+You can create user-defined tolerations to kernel modules to schedule selected KMM pods on a cordoned node. For example, during a device driver upgrade you can specify that a node is to be cordoned while, at the same time, you can run schedulable housekeeping pods that carry out the driver upgrades. 
```

Comment:
I am thinking of the following rewrites:

- 1. "...during a device driver upgrade you can cordon a node, at the same time, you can run housekeeping pods to upgrade a device driver."
- 2. "...during a device driver upgrade you cordon a node, at the same time, you can run housekeeping pods that perform the driver upgrades"

Reply:
Made this change (ii.)